Dockerfile-CI

```diff
@@ -10,6 +10,11 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update && \
 
 ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
 
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN unzip awscliv2.zip
+RUN ./aws/install
+RUN rm awscliv2.zip
+
```

Comment:
@ericsorenson Should we get the fix for the Java issues in with this change too? Otherwise it may break for some folks during hte interim.

Reply:
We didn't make any Java version changes. The DynamoDB version downgrade is in the next PR. As long as the cached version isn't deleted, running locally won't be an issue.

Reply:
> @julialeague regarding the Petition document link, Safari has a default popup blocker. The icon is on the right side of the address bar.

After clearing my cache, I have the same behavior as you. No worries!